microsoft-edge/webview2/how-to/self-hosting.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Self-host by deploying preview channels
+description: How to set up self-hosting by deploying using the WebView2 preview channels, which are the Insider preview channels of Microsoft Edge (Canary, Dev, and Beta).
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 04/18/2024
+---
+# Self-host by deploying preview channels
+
+_Self-hosting_ means deploying a preview WebView2 Runtime with your app.  This allows you and your development team to use your app daily with the preview version of the WebView2 Runtime, to find issues that will affect your particular WebView2 app, before the forthcoming changes reach the Evergreen WebView2 Runtime (Stable).
+
+The self-hosting approach provides real-world usage of your app with the latest preview WebView2 Runtime to catch any issues that may arise.  You should self-host your app in tandem with doing prerelease testing on your app, to provide a structured approach to testing your app with the latest version of the WebView2 Runtime.
+
+| Approach | Technique | Purpose/value |
+|---|---|---|
+| Prerelease testing | Automated testing on a preview version of WebView2 Runtime with your app. | Test your app with a preview version of the WebView2 Runtime to find issues before WebView2 upcoming changes reach the Stable Evergreen Runtime. |
```

Comment:
```suggestion
| Prerelease testing | Automated testing with a preview version of WebView2 Runtime. | Find issues before upcoming WebView2 changes reach the Stable Evergreen Runtime. |
```

Reply:
I'm handling.  Result, in both rows updated:
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/b32fef9b-f802-42b0-8556-ebeade923ea5)